### ISSUE TITLE
Windows builds

### ISSUE THREAD

**Issue by justin1121** [tf-encrypted/tf-encrypted#282]
Soon we will be building specific python wheel packages for linux and macos. To complete the triumvirate we'll need to support windows as well. As of now circleci does not support windows builds so we will need to come up with another solution. It looks like travis ci has just launched [windows builds](https://blog.travis-ci.com/2018-10-11-windows-early-release) so that is one solution.

**Comment by mortendahl** [comment]
Would be nice to have. Should we consider giving Travis a try? (my experience with Travis has been good but not a poweruser by any means)

**Comment by bendecoste** [comment]
I have used travis as well as circle ci, I don't really consider one better than the other.

**Comment by mortendahl** [comment]
We have seen a few OM users on Windows already, suspect there are several more out there. Started looking into how we could support development, including testing through eg Parallels.

**Comment by mortendahl** [comment]
@justin1121 you connected with Circle CI re their upcoming [early access for Windows](https://circleci.com/get-early-access-win/)?

**Comment by justin1121** [comment]
Yeah. Said I'd hear back in a week or two. The guy I talked to was just about to go on vacation.

**Comment by justin1121** [comment]
We have access to windows on circle-ci now. Here's some info on how to set it up. https://github.com/CircleCI-Public/windows-preview-docs

**Comment by mortendahl** [comment]
@jvmancuso and @yanndupis, any impression of how popular Windows is in DS environments? I haven't see it a lot in my dev roles.

Evidence in favour of supporting Windows:
- We've seen a few PySyft tickets already.
- Still very popular overall: [78% of worldwide users are on Windows](http://gs.statcounter.com/os-market-share/desktop/worldwide).
- Popular among visitors of tf-encrypted.io: [25% over last month, but sample size](https://analytics.google.com/analytics/web/).

Things in favour of not supporting Windows:
- Our thinking seems heavily adapted to unix environments, perhaps since none of us run Windows on a daily basis. We'd have to start remembering to represent file paths abstractly etc etc.
- Not sure many of us have (up-to-date) experience developing for and on Windows, meaning there's a learning curve.
- Build processes etc would have to be adapted to work on Windows.

**Comment by mortendahl** [comment]
@yanndupis could you add your note on experience from Deloitte?
